articles/custom_storage_location.md

```diff
@@ -49,6 +49,17 @@ To allow authorized Synapse users to upload data to your bucket set read-write p
             "Effect": "Allow",
             "Resource": "arn:aws:s3:::thisisthenameofmybucket/*",
             "Principal": { "AWS": "325565585839" }
+        },
+        {
+            "Action": [ "s3:PutObject" ],
+            "Effect": "Deny",
+            "Resource": "arn:aws:s3:::thisisthenameofmybucket/*",
+            "Principal": { "AWS": "325565585839" },
```

Comment:
This would enforcing the `bucket-owner-full-control` canned ACL for the Synapse AWS account. You probably don't want to do this. From what we can tell (still waiting for a confirmation [here](https://sagebionetworks.jira.com/browse/IT-1128?focusedCommentId=126974)), Synapse uploads data files to the custom bucket using `bucket-owner-full-control` whereas previews [are not](https://htanworkspace.slack.com/archives/C01RJ62NP7Y/p1616432929028000). I expect this would break Synapse's ability to generate previews and result in unexpected behavior. 

Here, the `Principal` would be any user who is given direct upload access to the bucket. 

/cc @brucehoff

Reply:
@BrunoGrandePhD You are exactly right.  We need to do https://sagebionetworks.jira.com/browse/PLFM-6663 before changing buckets to require `bucket-owner-full-control`.

Reply:
The policy doesn't need to deny Synapse (`325565585839`) in this way because we can make sure that Synapse adds the necessary ACL.  However it would be helpful if the policy denied all _other_ principals that the user might admit from uploading without the ACL.  Such a condition should only apply on to principals who are outside of the bucket owner's account since when principals *in* the bucket owner's account upload, the bucket owner will own the objects.